h5p_proplist_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package hdf5
+
+import (
+	"errors"
+	"math"
+	"os"
+	"testing"
+)
+
+/**
+ * These test cases are based on the h5_cmprss.c by The HDF Group.
+ * https://support.hdfgroup.org/HDF5/examples/intro.html#c
+ */
+
+func TestDeflate(t *testing.T) {
+	DisplayErrors(true)
+	fn, dsn, dims := "cmprss_deflate.h5", "dset_cmpress", []uint{1000, 1000}
+	defer DisplayErrors(false)
```

Comment:
I would move that `defer` just after the `DisplayErrors(true)` line.

Reply:
Fixed